### DISCUSSION TITLE
Selecting plot size for 2 2TB NVMEs

### DISCUSSION THREAD

**Discussion by cpereyra5199** [Chia-Network/chia-blockchain#4998] [Q&A]
Hello, I apologize ahead of time for the noob question.  I am in the process of setting up my chia miner using 2 2TB nvmes, which from what I have been reading should be able to support a total of 10 plots.  5 per NVME.  What I am not 100% sure of is what plot size I should be selecting.  I see the temporary space numbers for each k value, but not sure if that is per plot count of for the total plot count.  Could some provide some detail on which plot size I should be picking for my setup.  I saw the spreadsheet with different user setups but I don't see anywhere on the sheet where it specifies the plot size each user used.  Again apologies for the newbie questions!

Thank you

**Comment by padik11** [comment]
(of size 101.4GiB, k=32)

**Comment by moshenskyDV** [comment]
The sizes in table provided for each plot. So if you want to do 10 k32 in parallel you should have 256GB (239 GiB) x 10 = 2560 GB of temporary storage.

**Reply by cpereyra5199** [reply]
Does that theoretically mean I should be running more than 10 plots with 2 2TB nvmes at k32?  Or should I be running k33+ Plots?

thank you for the info

**Reply by moshenskyDV** [reply]
There is no big difference to generate 2xK32 or 1xK33. Explanation: [https://www.youtube.com/watch?v=knNPLoX5Jto&t](https://www.youtube.com/watch?v=knNPLoX5Jto&t)

You have a big amount of SSD memory, if you have 32gb RAM and 12+ threads on processor you can easily run a 10 plotting processes in parallel. There are also a lot of investigations on youtube how much RAM and threads you should spend for each plotting process.
